CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -15,29 +15,25 @@ public class HarvestGateway : IHarvestDeveloperRetriever, ITimeSheetRetriever
         private readonly string _token;
         private readonly string _accountId;
         private readonly string _userAgent;
-        private readonly List<string> _listOfDeveloperRoles = new List<string>
-        {
-            "Software Engineer",
-            "Senior Software Engineer",
-            "Senior Engineer",
-            "Lead Engineer",
-            "Delivery Manager",
-            "SRE",
-            "Consultant",
-            "Delivery Principal"
-        };
-
+        private readonly string[] _listOfDeveloperRoles;
```

Comment:
```suggestion
        private readonly string[] _developerRoles;
```
I think the `listOf` is unnecessary.

Reply:
Resolved here https://github.com/chubberlisk/crypto_tech/pull/31/commits/75edcdbf331cea21af64eb290e3d616733d02407.